File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/ExecuteHistoryBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.util.Date;

@Data
public class ExecuteHistoryBean {
    private String sqlInfo;
    private String dbName;
    private Long duration;
    private Date createTime;
    private String creator;
    private String params;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/ExecuteHistoryBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.util.*

@AllOpen
class ExecuteHistoryBean {
    var sqlInfo: String? = null
    var dbName: String? = null
    var duration: Long? = null
    var createTime: Date? = null
    var creator: String? = null
    var params: String? = null
}
